### PR TITLE
E2E: remove cli fixtures in favor of e2e

### PR DESCRIPTION
**Pull request by tooppaaa**

### CODE REVIEW

**Comment by tooppaaa** [comment]
@ndelangen if this is all green, should we proceed like this and forgot failing tests from the other PR ?

**Review by ndelangen — APPROVED** [review]

File: scripts/run-e2e-config.ts [hunk 1]
```diff
@@ -19,6 +19,14 @@ export const angular: Parameters = {
   additionalDeps: ['react', 'react-dom'],
 };
 
+export const angularv7: Parameters = {
+  ...baseAngular,
+  version: 'v7-lts',
+  additionalDeps: [...baseAngular.additionalDeps, 'core-js'],
```
Comment (gaetanmaisse) [inline scripts/run-e2e-config.ts:31]: I'm wondering why `core-js` is needed, can you give me some hints?

**Review by gaetanmaisse — APPROVED** [review]

**Comment by gaetanmaisse** [comment]
Just added 2 commits to fix Yarn 2 tests run, it was failing because we were trying to init project with a name invalid for NPM. Everything should be 🟢 .

Good job @tooppaaa 💪 👏

File: scripts/run-e2e-config.ts (hunk 1 above)
Comment (tooppaaa) [reply scripts/run-e2e-config.ts:31]: It's not "needed", but v7 (didn't checked above) installs core-js v2 which cause some issues with storybook

**Pull request merged**